### ISSUE TITLE
Seeking Collaborators for DayZ Epoch

### ISSUE THREAD

**Issue by vbawol** [vbawol/DayZ-Epoch#1660]
Since November 2012 the source code for DayZ Epoch as been available over the years we have had many contributors to the project. Most are no longer active however, and the task of maintaining DayZ: Epoch is monumental for any one person. 

If you want to become a Collaborator on the DayZ Epoch Project you must:
- Have a references showing some of your work.
- Be able to compile, test, and merge your changes on your own.

If some of you guys are willing to put some work in, I am willing pack and push at least one more update for DayZ Epoch.

**Notes**
The EOL (End Of Life) patch for Arma 2 will come out soon and is already on the beta branch for testing. This update will inevitably break something and DayZ Epoch may need need updates.  https://twitter.com/FoltynD/status/600680145314140160

**Comment by Gr8z** [comment]
I am here to help Awol. Was waiting for this update for a long time.

**Comment by slunski** [comment]
Also wanting to help. Also waiting for *.2.

What about packing-in some Overwatch-included mods ? Admins
should have that option (eg. mk16/17, more cars) even while
sticking to Epoch feel. It's just one line per mod in config (even if not
enabled by default)...

**Comment by deadlokd** [comment]
let me know what I can do to help.. sorry I haven't been around that much
as I am still in N.Y.. but let me know if I can help with anything.

**Comment by vbawol** [comment]
[11:27:01 AM | Edited 11:27:18 AM] David Foltyn: new update for OA beta is out, more likely final in terms of core unless you got something new (bugs)
any futher updates are bonus to actual state

**Comment by Adaptivity** [comment]
So, who will takeover Epoch? Do you have somebody in mind @vbawol?

**Comment by fr1nk** [comment]
I'm willing to help test if there's a server available at some point.

**Comment by Windmolders** [comment]
Forked, will loop over the issues and do pull requests.

**Comment by vbawol** [comment]
@Windmolders has been added as a collaborator so he may merge pull requests.

**Comment by icomrade** [comment]
Hey guys, I'm willing to lend a hand around here again. I've set up a test server @ **37.59.229.103** port **2402** the server name is "Epoch 1.0.6 Unofficial Test Server" and the password is **epochtest106**

I will update the files on the test server as often as needed but the quickest method would be to set up your own test server on your local machine and use the test server I have set up for extended testing.

I will upload the a ZIP of the client .PBO files the server is running shortly. the ZIP will NOT contain all necessary files, you will need to copy an existing Epoch 1.0.5.1 folder and overwrite the files with the ones I uploaded.

**Comment by Gr8z** [comment]
Don't Bother, This project has been abandoned by the developers.

**Comment by icomrade** [comment]
I'm aware of that, but a sign of someone interested in the project might bring some people willing to lend a hand. Awol doesn't seem against this becoming a community project either. if it doesn't there's no loss. I'm obviously not looking to takeover this repo by myself, so I don't see lending a hand and simply updating a test server once or twice a week as much bother.

**Comment by Windmolders** [comment]
It's nice to see interessent in the matter. Me myself and I might be helping again in the future aswell, been extremely bussy as i promoted to teamlead and do some work on the sides. But as soon as i can get my hands abit back on arma, I'm happy to help :).

I can assist quickly for accepting merge request and stuff if you need it.

**Comment by deadlokd** [comment]
nice job, thanks Zupa. I am also setting up a new rig. should be ready in
next couple of weeks and then i can get back on the project. Glad to see
theres still some people inspired by the game. Good Hunting.

**Comment by vbawol** [comment]
This project has been open for anyone to fork since its inception. The work is released so that other people can build upon it and hopefully contribute back to the project. 

@icomrade Sounds good, please let me know if there is anything you need assistance with. 

@Gr8z, @icomrade is a contributing developer on the project I think he knows the status of the project.

From what I have heard the next near EOL Arma 2 patch is due out soon:
https://forums.bistudio.com/topic/186695-a2a2oa-patch-18122015/

**Comment by ebaydayz** [comment]
@icomrade Your test server appears to be down atm.

**Comment by icomrade** [comment]
@ebaydayz Thanks for pointing out the code pbo was missing. Considering the lack of interest since December I took the server down since I'm working on putting up an ArmA 3 server. However, I'll set it up again provided there's some interest in the project still.

**Comment by vbawol** [comment]
@ebaydayz was added as a collaborator, thanks joining in the fun! Please make sure you guys collaborate as much as possible about your changes and please do let me know if you need any assistance.

**Comment by vbawol** [comment]
I made a discord channel if any of you guys want to ask me any questions or just talk with each other please join: https://discord.gg/0k4ynDDCsnQ1De32

**Comment by ebaydayz** [comment]
Thanks. I will try to have at least one additional person review and test any significant changes I make before committing. It will be tough since we don't have a lot of active people here. However, it is obvious from the player numbers there are still enough people who care about the mod to make it worthwhile. 

The A2OA EoL patch is out now. It appears to have caused minimal breakage apart from MedBox0 capacity and some "No owner" RPT spam. Both of those are already fixed in the next beta patch, but it's not clear when exactly it will be released to stable.

I'll be in the discord while I'm working here. If I'm AFK just leave me a message or PM me on the forums.

**Comment by vbawol** [comment]
FYI, I have transfered the github to EpochModTeam org on github. old links will be redirected but the new link is now: https://github.com/EpochModTeam/DayZ-Epoch

**Comment by Adaptivity** [comment]
Glad to see you continued work on the mod guys @ebaydayz @icomrade. Hope to see an update to latest DayZ mod code in future. Thank you a lot.

**Comment by includes08** [comment]
@vbawol @Windmolders  if you want i can run a test server for the community and keep it up to date everytime there is a push. If interested :)?

**Comment by icomrade** [comment]
I have a dedicated server, but it's being updated to server 2016 so I haven't updated it recently (I've kept it down for the past couple days), I will update it today since the host literally just sent me an email about the setup being finished.

We encourage everyone to host a test server and send logs and or feedback to us

**Comment by includes08** [comment]
@icomrade ah, well i have enough space left :) so i'm not that bothered. I love arma 2, and i don't mind maintaining it. ^^ so just let me know if its okay :P

**Comment by icomrade** [comment]
I've been having some trouble getting the server up on windows server 2016. It appears the server doesn't load .dll files anymore, either from the mod folders or directly in the A2OA root.

**Comment by oiad** [comment]
I hear the references to unblocking DLLs in quite a lot of howtos, could that be the case? I think they suggest right click DLL and goto unblock

**Comment by icomrade** [comment]
First thing I checked, it's not as big of an issue in 2016 as it was in 2012.

**Comment by ndavalos** [comment]
@icomrade folder permissions and c++ runtimes. Set the Arma 2 folder to full access to group "Everyone" or alternately make sure you're running it as an administrator (log in with an account that is in the administrator group, don't bother with "run as"). Make sure you got all your c++ runtimes installed as well. I would also not run it from the default steam install location (%program files%). I would install steam elsewhere like directly on the root of the drive (or just run the server from a copy of OA in the root, but copy the A2 "addons" folder into it if you do that). I installed steam in the root of the drive in order to avoid folder permissions in 2012. I actually have had no issue with 2012 R2 and running Arma, it just worked right out of the box for the most part (after installing the appropriate runtimes).

I have not played with 2016 yet, you may need to sign your dlls now. That's always a possibility. Also look in the application, system and security windows event logs for errors, it should be logging the fact that it's preventing hive from being loaded into memory. Usually you only have to sign drivers, but they may have made a change to make malware harder to infect a system by requiring dlls and executables to be signed.

Also set compatibility for the arma executables if none of this works. It could be something new with running x86 code on x64 in 2016. One other thing to check is DEP, you could switch it to essential windows services and programs only or add exceptions for the arma server executable and hive. I highly doubt the dll is being executed or loaded in the default heap or stack, but who knows, it's Arma and this version is pretty old...

Most likely it's folder permissions. Check the permissions of the user logged in, right click folder >> properties >> security >> advanced >> effective access >> pick user >> view effective access.

**Comment by icomrade** [comment]
was missing the 2010 redistributable... thanks for the suggestions,

I swear I installed every last one, I'm surprised that I wasn't getting any errors

**Comment by ndavalos** [comment]
Yeah C++ redistributables happens to me whenever I install a new mod somewhere, I forget at least one and spend 2 days trying to figure out what went wrong :P
